build_deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+IMAGE_BUILD="ekidd/rust-musl-builder:1.30.1"
+
+IMAGE="quay.io/app-sre/cincinnati"
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+DOCKERFILE_DEPLOY="dist/Dockerfile"
+
+# clean up
+rm -rf target
```

Comment:
Is this necessary in a CI environment? Need to clarify if this is also meant to be used on developer machines.

Reply:
`cargo clean` can do that, but why do we need to manually clean this up?

Reply:
I just don't know what will happen when I will rerun the build, will it automatically clean target?
I think of it as similar to `mvn clean install` like clean up first and then new target will be generated once build is complete.
should I replace this with `cargo clean`?

Reply:
Rerunning the build will produce a new binary if the source has changed. This has worked reliably for me during development so I wouldn't worry about it for local development.
I'm still trying to figure out if this script is supposed to be used on a local development machine or not ;-) If that's the case we probably need to adapt it for a couple of things:
* Don't remove the bulid cache (which happens on the line we're currently commenting on)
* Don't try to push the image
* Swtich from a hardcoded `/home/rust/src` (see above)

Reply:
it's up to you use these scripts for local dev or not, from my experience some teams are doing that.

Reply:
I've removed clean up from the script